### PR TITLE
Add social links [GCI2019 TASK]

### PR DESCRIPTION
**Pull request by makoteq**
Fix for https://github.com/OpnTec/vn.opentechsummit.asia/issues/43
**Preview link**: https://makoteq.github.io/vn.opentechsummit.asia/

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-vn.opentechsummit.asia
```
Comment (nikit19) [inline CNAME:1]: Please revert this change

**Review by nikit19 — CHANGES_REQUESTED** [review]

**Comment by makoteq** [comment]
Now its ok? @nikit19

**Comment by mariobehling** [comment]
Thank you.
